### PR TITLE
refactor(frontend): use proxy subchart

### PR DESCRIPTION
**Pull request by fdlk**
Reminder: we must update the value keys set in the molgenis-frontend Jenkinsfile when it deploys this chart.

### CODE REVIEW

File: charts/molgenis-frontend/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "8.1"
 description: MOLGENIS Helm chart for the frontend
 name: molgenis-frontend
-version: 1.0.2
+version: 1.0.3
```
Comment (fdlk) [inline charts/molgenis-frontend/Chart.yaml]: Bit unsure, if the key names are part of the API, this is a breaking change.

Comment (sidohaakma) [reply charts/molgenis-frontend/Chart.yaml]: No, I would leave it a fix.

File: charts/molgenis-frontend/questions.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ questions:
   type: string
   required: true
   group: "Provisioning"
-- variable: backend.url
+- variable: molgenis-proxy.url
   label: Backend
   default: "https://latest.test.molgenis.org"
```
Comment (sidohaakma) [inline charts/molgenis-frontend/questions.yml]: Switch to master.dev.molgenis.org just like in values.yaml?

Comment (fdlk) [reply charts/molgenis-frontend/questions.yml]: It should be on same cluster. Is inconsistent now, I'll fix it.

**Pull request merged**